package.json

```diff
@@ -93,10 +95,11 @@
     "request": "2.83.0",
     "scooter": "5.0.0",
     "stripe": "5.1.1",
-    "style-loader": "0.13.0",
+    "style-loader": "0.18.2",
     "throng": "4.0.0",
-    "webpack": "1.12.9",
-    "whatwg-fetch": "0.10.1"
+    "webpack": "3.2.0",
```

Comment:
mainly what happened here is that snyk wanted to bump webpack to 3.2.0, but I had to update a bunch of other stuff to keep the build working.